### ISSUE TITLE
Model not added to solution

### ISSUE THREAD

**Issue by MrFanciful** [dotnet/machinelearning-modelbuilder#1611]
**System Information (please complete the following information):**
 - Model Builder or CLI Version: Model Builder
 - Visual Studio Version (if applicable): 2019

**Describe the bug**
- On which page of the tutorial did you run into an issue (URL): https://dotnet.microsoft.com/learn/ml-dotnet/get-started-tutorial/generate-code
- Clear description of the problem: VS not adding myMLAppML.Model to project

**To Reproduce**
Clicking "Add to solution" only adds the console app with a different name from the step in the tutorial. I only get MLModel1_ConsoleApp1 added to the solution. Subsequent code in next step doesn't work without Model.

**Expected behavior**
myMLAppML.ConsoleApp and myMLAppML.Model added to solution

**Comment by zewditu** [comment]
@MrFanciful  yes you are right, the documentation for the tutorial has to be updated. Now the sample projects use a different  default name pattern-> your ModelName-ConsoleApp1 ,  your ModelName-ConsoleApp2 ...

**Comment by zewditu** [comment]
@MrFanciful sorry for the inconvenience as the temporary solution you can refer to this recent release blog post 
https://devblogs.microsoft.com/dotnet/ml-net-june-updates-model-builder/#model-builder-updates

**Comment by briacht** [comment]
I will work on getting the website tutorial updated this week!

**Comment by briacht** [comment]
Just put in a PR for dotnet website to update the tutorial
